frontend/gist-frontend/src/Components/LoginPageComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import LoginOptionsComponent from "./LoginOptionsComponent";
+import AboutGistComponent from "./AboutGistComponent";
+
+import { Box, Grommet } from "grommet";
+
+interface ILoginPageComponentProps {
+  theme: any;
+}
+
+export default class LoginPageComponent extends React.Component<
+  ILoginPageComponentProps
+> {
+  render() {
+    return (
+      <Grommet theme={this.props.theme} full>
+        <Box fill background="brand">
```

Comment:
played around with a theme if you wanna check it out

`<Box
          fill
          background="linear-gradient(160deg,#a8f4d3,#7de3e1,#6cceea,#7cb5e6,#94a7e1,#aba5e5,#c5a1e4,#de9ddd)"
        >`